common/templates/emails/new_enrollment_email_body.txt

```diff
@@ -0,0 +1,5 @@
+Enrollment received for:
+learner: ${user.username} (${user.get_full_name()})
+course: ${course_url}
```

Comment:
Not sure of the UX involved here with a raw course URL being pasted in. Depending on that, this may have to have an HTML alternative where 'Course' is clickable or something.

Reply:
I have sent me a test e-mail in text/plain and verified that e.g. gmail makes URLs clickable. As long as we don't need design, text/plain is fine for URLs.
  I also don't know the UX requirements, maybe this needs a logo, header, footer etc. I expect the client or edx to come back with changes to this template.